src/Security/Http/AnonymousFollowerAuthenticationSuccessHandler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace AppBundle\Security\Http;
+
+use AppBundle\Security\Http\Session\AnonymousFollowerSession;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Authentication\Token\AnonymousToken;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Http\Authentication\DefaultAuthenticationSuccessHandler;
+use Symfony\Component\Security\Http\HttpUtils;
+
+class AnonymousFollowerAuthenticationSuccessHandler extends DefaultAuthenticationSuccessHandler
+{
+    private $anonymousFollowerSession;
+
+    public function __construct(HttpUtils $httpUtils, array $options = [], AnonymousFollowerSession $anonymousFollowerSession)
+    {
+        parent::__construct($httpUtils, $options);
+
+        $this->anonymousFollowerSession = $anonymousFollowerSession;
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
I found other occurrences.

Anyway, I would hardly recommend to keep such php-doc (as we do for `@return someClass[]` while using `: array`).
Since it is a hint that a method overrides a parent definition or implements an interface. It helps both understandability while reviewing and the discoverability of the code for new comers.